src/main/kotlin/lotto/vo/ManualLottos.kt

```diff
@@ -0,0 +1,18 @@
+package lotto.vo
+
+class ManualLottos(private val price: Long, private val buyCount: Long) {
```

Comment:
p2; price는 멤버 변수일 필요는 없어보여요.

```suggestion
class ManualLottos(price: Long, private val buyCount: Long) {
```